components/homepage/sections/Solutions.tsx

```diff
@@ -33,10 +31,42 @@ const solutions: SolutionCardProps[] = [
       "Add powerful web3 features to your Shopify storefront enabling tokengated commerce, NFT loyalty programs, digital collectible sales, and more.",
     img: "/assets/landingpage/CommerceKit.png",
     partnerIcon: "/assets/landingpage/icons/Shopify.svg",
-    // arrowIcon: "/assets/landingpage/icons/arrow-pink.svg",
     href: "/solutions/commerce",
     hoverBorderColor: "hsl(309deg 54% 81% / 15%)",
   },
+  {
+    title: "Minting",
+    gradient: "linear-gradient(90deg, #8e0eff, #16bdf0)",
+    description: (
+      <>
+        An all-in-one toolkit to build and mint NFTs. Create powerful NFT
+        experiences. Build your own NFT and Marketplace contracts with{" "}
+        <TrackedLink
+          color="blue.500"
+          isExternal
+          category="solutions"
+          label="contractkit"
+          href="https://portal.thirdweb.com/contractkit"
+        >
+          ContractKit
+        </TrackedLink>{" "}
+        or discover and deploy in 1-click via{" "}
+        <TrackedLink
+          color="blue.500"
```

Comment:
can we make these links white? I think the blue looks odd in these contexts, they should still stand out enough?